install/install_openadapt.ps1

```diff
@@ -70,205 +75,181 @@ function CheckCMDExists {
-    $exists = Test-Path -Path $pythonInstaller -PathType Leaf
-    if (!$exists) {
-        Write-Host "Failed to download python installer"
-        Cleanup
-        exit
-    }
-
-    Write-Host "Installing python, click 'Yes' if prompted for permission"
-    Start-Process -FilePath $pythonInstaller -Verb runAs -ArgumentList '/quiet', 'InstallAllUsers=0', 'PrependPath=1' -Wait
-
-    #Refresh Path Environment Variable
-    $env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path", "User")
-
-    # Make sure python is now available and the right version
-    if (CheckCMDExists($pythonCmd)) {
-        $res = Invoke-Expression "python -V"
-        if ($res -like $pythonVerStr) {
-            Remove-Item $pythonInstaller
-            return $pythonCmd
-        }
-    }
-
-    Write-Host "Error after installing python. Uninstalling, click 'Yes' if prompted for permission"
-    Start-Process -FilePath $pythonInstaller -Verb runAs -ArgumentList '/quiet', '/uninstall' -Wait
-    Remove-Item $pythonInstaller
-
-    # Stop OpenAdapt install
-    Cleanup
-    exit
-}
-
 function GetTesseractCMD {
     # Use tesseract alias if it exists
-    if (CheckCMDExists($tesseractCmd)) {
+    if (CheckCMDExists $tesseractCmd) {
         return $tesseractCmd
     }
 
+    # Check if tesseractPath exists and delete it if it does
+    if (Test-Path -Path $tesseractPath -PathType Container) {
+        Write-Host "Found Existing Old TesseractOCR, Deleting existing TesseractOCR folder"
+        # Delete the whole folder
+        Remove-Item $tesseractPath -Force -Recurse
+    }
+
     # Downlaod Tesseract OCR
     Write-Host "Downloading Tesseract OCR installer"
     $ProgressPreference = 'SilentlyContinue'
     Invoke-WebRequest -Uri $tesseractInstallerLoc -OutFile $tesseractInstaller
     $exists = Test-Path -Path $tesseractInstaller -PathType Leaf
     if (!$exists) {
-        Write-Host "Failed to download Tesseract OCR installer"
+        Write-Host "Failed to download Tesseract OCR installer" -ForegroundColor Red
         Cleanup
         exit
     }
 
     # Install the Tesseract OCR Setup exe (binary file)
-    Write-Host "Installing Tesseract OCR, click 'Yes' if prompted for permission"
+    Write-Host "Installing Tesseract OCR..."
     Start-Process -FilePath $tesseractInstaller -Verb runAs -ArgumentList "/S" -Wait
     Remove-Item $tesseractInstaller
 
     # Check if Tesseract OCR was installed
     if (Test-Path -Path $tesseractPath -PathType Container) {
-        Write-Host "TesseractOCR installation successful."
+        Write-Host "TesseractOCR installation successful." -ForegroundColor Green
     }
     else {
-        Write-Host "TesseractOCR installation failed."
+        Write-Host "TesseractOCR installation failed." -ForegroundColor Red
         Cleanup
         exit
     }
 
-    #Refresh Path Environment Variable
+    # Refresh Path Environment Variable
```

Comment:
Can GetTesseractCMD be simplified so Refresh Path Environment is called less often?

Reply:
I wanted to be extra carefult when dealing with editing PATH variables. 

So here is my strategy:
Refresh (before the adding to System PATH var)
Add to System PATH var
Refresh again (before the adding the User Account PATH var)
Refresh (again after adding, this one is unnecessary on thinking and can be removed

Reply:
Addressed in: [3217540](https://github.com/MLDSAI/OpenAdapt/pull/312/commits/3217540d6882f0a384f5790c4c02bba0fd8f1f64)

Reply:
Better idea: Addressed in: [50d06bd](https://github.com/MLDSAI/OpenAdapt/pull/312/commits/50d06bd93e905adc3791b4fb28b5a5393052497a)